tofhir-server/api.yaml

```diff
@@ -1,22 +1,38 @@
 openapi: 3.0.2
 info:
   description: "Lists the REST APIs provided by a toFHIR server"
-  version: "1.0.0"
+  version: "3.9.0"
```

Comment:
I believe this is the version of tofhir-server. It should be 1.1-SNAPSHOT or simply 1.1

Reply:
Okay, I am writing 1.1